src/index.tsx

```diff
@@ -94,9 +94,29 @@ function setupWebEvents(config: Config) {
 }
 
 function Loader() {
+    const config = useContext(ConfigContext);
+
+    const interfaceConfig = config.values.interface;
+    const theme = interfaceConfig.theme;
+    const backgroundColorOverride =
+        interfaceConfig.styleOverrides[theme ?? "light"]?.backgroundColor;
+
     return (
-        <div className="lds-ring">
-            <div></div><div></div><div></div><div></div>
+        <div
+            style={{
+                display: "flex",
+                justifyContent: "center",
+                alignItems: "center",
+                width: "100%",
+                height: "100%",
```

Comment:
Move this to `loader.css`.
Also spinner needs to be a bit brighter in dark theme for contrast.

Reply:
> Also spinner needs to be a bit brighter in dark theme for contrast.

So I'd need to move `border-color` from `loader.css` into `index.ts` and apply a brighter color depending on configured theme? Or is there a trick to keep that in the css file and still have it depend on the chosen theme?

Reply:
> move border-color from loader.css into index.ts and apply a brighter color depending on configured theme

Either that or use additional class for dark theme overriding the border-color.

Reply:
It seems so obvious now 😂
Anyway, let me know if you want me to change the implementation to use that additional class.